server.go

```diff
@@ -315,12 +315,16 @@ func (s *ExtensionManagerServer) Call(ctx context.Context, registry string, item
 func (s *ExtensionManagerServer) Shutdown(ctx context.Context) (err error) {
 	s.mutex.Lock()
 	defer s.mutex.Unlock()
-	stat, err := s.serverClient.DeregisterExtension(s.uuid)
-	err = errors.Wrap(err, "deregistering extension")
-	if err == nil && stat.Code != 0 {
-		err = errors.Errorf("status %d deregistering extension: %s", stat.Code, stat.Message)
+
+	if s.serverClient != nil {
+		var stat *osquery.ExtensionStatus
+		stat, err = s.serverClient.DeregisterExtension(s.uuid)
+		err = errors.Wrap(err, "deregistering extension")
+		if err == nil && stat.Code != 0 {
+			err = errors.Errorf("status %d deregistering extension: %s", stat.Code, stat.Message)
+		}
 	}
-	s.serverClient.Close()
```

Comment:
Don't we need to close ?

Reply:
Or is it closed [here](https://github.com/osquery/osquery-go/commit/dbeefc07d76e3edaf614647a20cc588c1b4f47e4#diff-366e46a40f6f60b4f7614eb0976bb51820364bf5ca6ccc4787eb49d7bdbef3e6R337)?

Reply:
Correct.

Reply:
Huh. This line looks like I should have removed it in #112